tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -133,7 +133,7 @@ class VerifiableConsumer(BackgroundThreadService):
         }
 
     def __init__(self, context, num_nodes, kafka, topic, group_id,
-                 max_messages=-1, session_timeout=30000, enable_autocommit=False,
+                 max_messages=-1, session_timeout_sec=30, enable_autocommit=False,
```

Comment:
Yeah, that's a good point. I think this change is justifiable though since it makes the unit of the session timeout explicit and follows the apparent naming convention in ducktape and the other test cases. The tests were already incorrectly passing `session_timeout` directly to `wait_until`, which would have been a harder mistake to make if the unit was clear.